transforms/no-vars.js

```diff
@@ -0,0 +1,79 @@
+/** 
+ * Transform all your `var`s to `let` and `const`.
+ *
+ * The only catch is it may not catch nested bodies that mutate anything beyond
+ * a single parent.
+ */
+
+
+module.exports = function(file, api) {
+  const j = api.jscodeshift;
+
+  // a quick and likely incomplete check if a variable has been mutated within
+  // a variables direct parent scope
+  var isMutated = (node) => {
+    var mutated = false;
+    j(node.parent)
+      .find(j.AssignmentExpression)
+      .forEach(n => {
+        if (node.value.declarations.some(d => d.id.name === n.value.left.name)) {
+          mutated = true;
+        }
+      });
+
+    j(node.parent)
+      .find(j.UpdateExpression)
+      .forEach(n => {
+        if (node.value.declarations.some(d => d.id.name === n.value.argument.name)) {
+          mutated = true;
+        }
+      });
+    return mutated;
+  }
+
+  var ast = j(file.source)
```

Comment:
Its up to you but in true jQuery fashion i call these vars `jRoot` so i can easily tell whats a collection.